test/test_libs.py

```diff
@@ -636,7 +636,7 @@ def test_vmas_spec_rollout(
         )
         for e in [env, wrapped]:
             e.set_seed(0)
-            check_env_specs(e)
+            check_env_specs(e, check_dtype=False)
```

Comment:
@matteobettini I had to turn this of because some float32 are float64 or the opposite. 
Worth having a look on your side. I don't think it's a torchrl issue but not sure about that

Reply:
Oh yes it was a vmas issue i fixed, as soon as i release the new release on pypi it should be fixed